apps/dispatch-service/src/app/services/sbatch/sbatch.service.ts

```diff
@@ -205,17 +206,21 @@ echo -e '${cyan}========================================== Downloading COMBINE/O
 
 echo -e ''
 echo -e '${cyan}=========================================== Executing COMBINE/OMEX archive ==========================================${nc}'
+TEMP_DIRNAME=$(mktemp --directory --tmpdir=/local)
+set +e
 srun --job-name="Execute-project" \
   singularity run \
-    --tmpdir /local \
     --bind ${workDirname}:/root \
-    --bind /local:/tmp \
-    "${allEnvVarsString}" \
+    --bind \${TEMP_DIRNAME}:/tmp \
+    ${allEnvVarsString} \
     ${simulatorImage} \
       -i '/root/${combineArchiveFilename}' \
       -o '/root/${outputsS3Subpath}'
-
-set +e
+executeArchiveStatus=$?
+rm -rf \${TEMP_DIRNAME}
+if [[ $executeArchiveStatus -ne 0 ]]; then
+  exit $executeArchiveStatus
```

Comment:
Will this prevent any logs from being uploaded if present in the event that the execution fails?